web/db/migrations/20260807010000_stripe_reconcile_cursor/migration.sql

```
@@ -0,0 +1,6 @@
+ALTER TABLE "stripe_subscriptions"
+  ADD COLUMN "last_reconciled_at" timestamp with time zone;
+
+CREATE INDEX "stripe_subscriptions_reconcile_cursor_idx"
+  ON "stripe_subscriptions"
+  ("last_reconciled_at" ASC NULLS FIRST, "id" ASC);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)